### PR TITLE
nodejs/week1 HW

### PR DESCRIPTION
**Pull request by nuha90**

### CODE REVIEW

**Comment by h09shais** [comment]
Hi Nuha!
Thank you for you W01 PR 👍 !
1.  File rename - Use cheap-meals.js instead of cheap_meals.js 
2. Have you tried running npm test? What you get :) ?

**Comment by nuha90** [comment]
i got this 
'jest' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! Test failed.  See above for more details.

File: nodejs/week1/Circle_calculator.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  }
+
+  getDiameter() {
+    return this.radius * 2;
+  }
+  getCircumference() {
+    return Math.PI * this.radius * 2;
+  }
+  getArea() {
+    return Math.PI * this.radius * this.radius;
```
Comment (kfazinic) [inline nodejs/week1/Circle_calculator.js:13]: You can also get `number` to the power of `x` as `Math.pow(number, x)` or `number**x`. 👍

File: nodejs/week1/src/backend/routes/cheap-meals.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const express = require("express");
+const router = express.Router();
```
Comment (kfazinic) [inline nodejs/week1/src/backend/routes/cheap-meals.js:2]: Well done for using `Router()` 👏

File: nodejs/week1/src/backend/routes/cheap-meals.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+router.get("/", async (request, response) => {
+  response.json(
+    meals
+      .map((meal) => {
+        return {
+          ...meal,
+          reviews: reviews.filter((review) => {
+            return review.mealId === meal.id;
+          }),
+        };
```
Comment (kfazinic) [inline nodejs/week1/src/backend/routes/cheap-meals.js:15]: 👏  great job for using destructuring assignment

**Review by kfazinic — APPROVED** [review]
Perfect! 👏 💯

**Pull request merged**